google-api-client/.repo-metadata.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "google-api-client",
```

Comment:
I think we should omit "name" from these libraries since they're not API clients connected to a specific API. The "name" seems to be the actual API name in the repo-metadata files in google-cloud-ruby.